src/app/components/Button/hooks/useButtonClasses.ts

```diff
@@ -29,6 +30,8 @@ export const useButtonClasses = ({
     "secondary-bordered": "border border-white",
     transparent:
       "border-transparent bg-transparent focus-visible:bg-transparent focus-visible:border-theme-primary-700",
+    "primary-bordered":
```

Comment:
there is already a button like this for the navbar, can you reuse that? or rename the navbar buttons to use a `primary-bordered` button as well if that's better suited 🤔 

also I don't think we should have a button variant that only targets the dark mode; instead it should be a button that has a certain design in light and in dark so it doesn't have to switch classes like that.

also, the secondary button is now equal to the primary one in dark mode. It will be a good idea to see if we can remove that as well as there is no need for a "secondary" option otherwise for dark mode, which is weird 🤔 

is there a button design somewhere for light/dark mode that highlights the types of buttons that are used on demo? Otherwise it will be good to have an overview like that so we know how buttons translate between themes